nin/dasBoot/RootNode.js

```diff
@@ -0,0 +1,34 @@
+(function(NIN) {
+  'use strict';
+
+  class RootNode extends NIN.Node {
+    constructor(id, options) {
+      super(id, {
+        inputs: {screen: new NIN.TextureInput()}
+      });
+
+      this.camera = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1); 
+      this.scene = new THREE.Scene();
+      this.quad = new THREE.Mesh(new THREE.PlaneBufferGeometry(2, 2), null);
+      this.scene.add(this.quad);
+      this.material = new THREE.MeshBasicMaterial();
+    }
+
+    render(renderer) {
+      if(renderer.overrideToScreenTexture) {
```

Comment:
Maybe not all, since perhaps not all nodes should be previewed by being rendered by the THREEJS renderer. I chose this as a hacky entrypoint because it was easy for textures.